pkg/crc/machine/machine.go

```diff
@@ -9,9 +9,9 @@ import (
 	"github.com/code-ready/crc/pkg/crc/logging"
 	"github.com/code-ready/crc/pkg/crc/machine/bundle"
 	"github.com/code-ready/crc/pkg/crc/network"
-	"github.com/code-ready/machine/libmachine"
+	"github.com/code-ready/crc/pkg/libmachine"
+	"github.com/code-ready/crc/pkg/libmachine/host"
 	"github.com/code-ready/machine/libmachine/drivers"
-	"github.com/code-ready/machine/libmachine/host"
 	"github.com/code-ready/machine/libmachine/log"
```

Comment:
What would prevent this to become `log "github.com/code-ready/crc/pkg/crc/logging" ?

Reply:
Not much. I have a PR almost ready after this one to remove the machine logging and use traditional logrus.

Reply:
👍

Reply:
Do you want to address that, or push another PR to finalize? I'd rather not keep PRs hanging around, as this causes unnecessary difficult rebases, etc.

Reply:
I have to change machine before propagating it here in crc :/